23/09/2019 15:40:42.255 [main] ERROR Testcase.RegressionTest - Auth creation failed
23/09/2019 15:40:42.261 [main] ERROR Testcase.RegressionTest - {"_v":"18.8","fault":{"arguments":{"origin":"https://www.dummyapi.com"},"type":"UnauthorizedOriginException","message":"The origin 'https://www.dummyapi.com' is unauthorized."}}
